spec/support/helpers/controller_helper.rb

```diff
@@ -1,6 +1,16 @@
 module ControllerHelper
-  def stub_logged_in_user(user)
-    identity = SignonIdentity.new(user, full_name: FFaker::Name.name, profile_url: '', logout_url: '')
-    allow(controller).to receive(:sso_identity).and_return(identity)
+  def stub_logged_in_user(user, estate, available_estates: [estate])
```

Comment:
Being pedantic, this method is no longer accurately named. It doesn't stub the login, it actually logs in the user.